build_scripts/clean_sp.py

```
@@ -0,0 +1,155 @@
+"""
+Strip raw CodeRabbit HTML noise from .claude/S&P.md.
+
+Two modes:
+  python build_scripts/clean_sp.py          — clean .claude/S&P.md in place
+  python build_scripts/clean_sp.py --stdin  — read review body from stdin, write cleaned to stdout
+
+The script handles:
+  - </blockquote></details> on a single line (multiple closers per line)
+  - Inline text before </details> is preserved (not dropped with the tag)
+  - Counters reset only at genuine S&P top-level entry headings (## YYYY-MM-DD ...)
+  - Noise blocks fully removed: 🤖 Prompt, 🪄 Autofix, ℹ️ Review info, etc.
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF002)